### PR TITLE
Add person pivots and refactor is favorited

### PR DESCRIPTION
**Pull request by mattleff**
Fixes https://trello.com/c/a7dBymac/57-person-pivot-details.

image

### CODE REVIEW

**Review by narthur — APPROVED** [review]
Great work!

File: src/lib/routes.ts [hunk 1]
```diff
@@ -1,58 +1,59 @@
+import { Scalars } from './generated/graphql';
+
 const slug = (s: string): string => s.replace(/\s/g, '-').toLowerCase();
 
 export const makePaginationRoute = (
 	base: string,
-	page: number | string
+	page: Scalars['ID']
```
[the remark this answers is not in the document]
Comment (mattleff) [reply src/lib/routes.ts]: Good point: cfe0085

File: src/pages/[language]/bibles/[id]/[book].tsx [hunk 2]
```diff
@@ -40,7 +40,7 @@ export async function getStaticPaths(): Promise<StaticPaths> {
 	const books = bibles.map((b) => b.books).flat();
 	const bookIds = books.map((b) => b.id);
 	const pathSets = languageRoutes.map((r) =>
-		bookIds.map((id) => makeBibleBookRoute(r, id))
+		bookIds.map((id) => makeBibleBookRoute(r, id + ''))
```
[the remark this answers is not in the document]
Comment (mattleff) [reply src/pages/[language]/bibles/[id]/[book].tsx]: Yup: cfe0085

File: src/styles/styles.scss [hunk 3]
```diff
@@ -99,6 +96,15 @@ h6 {
 	line-height: 12px;
 }
 
+@media (min-width: $breakpoint-lg) {
+	h1 {
+		font-size: 44px;
+	}
+	h1 {
```
[the remark this answers is not in the document]
Comment (mattleff) [reply src/styles/styles.scss]: Yup: cfe0085

**Pull request merged**